cfme/tests/services/test_catalog_item.py

```diff
@@ -147,35 +147,64 @@ def test_edit_tags(catalog_item):
     catalog_item.remove_tag(tag)
 
 
-@pytest.mark.skip('Catalog items are converted to collections. Refactoring is required')
-@pytest.mark.meta(blockers=[BZ(1531512, forced_streams=["5.8", "5.9", "upstream"])])
-def test_catalog_item_duplicate_name(catalog_item):
+@pytest.mark.meta(blockers=[BZ(1531512, forced_streams=["5.9", "5.10"])])
```

Comment:
Question:
Is this issue fixed on `upstream`?

Reply:
No, but we don't test our automation on upstream.

Reply:
We certainly do, and all framework and automation components must be written to run against MIQ.